src/index.scss

```diff
@@ -0,0 +1,227 @@
+@mixin center {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+}
+
+body {
+  box-sizing: border-box;
+  margin: 0;
+  color: #ffeead;
+  background: #ffeead;
+}
+
+.containter {
+  display: grid;
+  grid-template-areas:
+    'logo     header'
+    'sidebar  content'
+    'sidebar  content'
+    'footer   footer';
+  grid-template-rows: 50px minmax(calc(50vh - 75px), 100%) minmax(calc(50vh - 75px), 100%) 50px;
```

Comment:
Try to replace minmax(calc(50vh - 75px), 100%) with auto. It should work too. But with less code
50px auto auto 50px

Reply:
Yes, but in IE `auto` works wrong
https://css-tricks.com/css-grid-in-ie-css-grid-and-the-new-autoprefixer/#article-header-id-15

Reply:
Cool, nice notice. Then keep it like that